### PR TITLE
Standalone processors under the hood (WASM)

### PR DESCRIPTION
**Pull request by lovromazgon**
Part of https://github.com/ConduitIO/conduit/issues/1422.

### CODE REVIEW

File: docs/processors/standalone/wasm.mdx [hunk 1]
```diff
@@ -0,0 +1,229 @@
+---
+sidebar_position: 2
+---
+
+# How it works under the hood (Wasm)
```
Comment (raulb) [inline docs/processors/standalone/wasm.mdx:5]: I'm wondering if we should simply could call it "How it works" and then explain in the document why using Wasm in the first place.

I think that could still valuable to explain the versatility this brings.

**Review by raulb — APPROVED** [review]
Looks great. I love seeing these going more in depth. I only have one question/comment on maybe adding an explanation of why wasm, or the benefits this brings us while also maybe tweaking the title of this page.

@lovromazgon what do you think?

File: docs/processors/standalone/wasm.mdx (hunk 1 above)
Comment (lovromazgon) [reply docs/processors/standalone/wasm.mdx:5]: I was expecting a comment about the title, I'm not too happy with it myself. `How it works` sounds better, the `under the hood` part is kinda redundant. I'm wondering if there's a way to include `Wasm` in the title without making it too long? 🤔 I feel like with that in the title you already get an idea that standalone processors are Wasm modules, without even opening the page.

I'll add an explanation on top why Wasm is a good choice 👍

Comment (lovromazgon) [reply docs/processors/standalone/wasm.mdx]: Done!

File: docs/processors/standalone/wasm.mdx [hunk 2]
```diff
@@ -0,0 +1,259 @@
+---
+sidebar_position: 2
+---
+
+# How it works
+
+Here we will describe how Conduit runs standalone processors under the hood.
+This is useful to understand when writing a processor SDK.
+
+## Why WebAssembly?
```
Comment (raulb) [inline docs/processors/standalone/wasm.mdx:10]: Making this suggestion to be consistent with we express these in our docs (e.g.: "How it works" vs "How does it work?")

```suggestion
## Why WebAssembly
```

**Review by raulb — APPROVED** [review]
Approved with just a minor suggestion. I really liked the introduction about why we chose to use wasm for our processors, and the options these provide. Nice work.

File: docs/processors/standalone/wasm.mdx [hunk 3]
```diff
@@ -0,0 +1,259 @@
+---
```
Comment (raulb) [inline docs/processors/standalone/wasm.mdx:1]: @lovromazgon I just realized it'd be probably better if we renamed this page to `how-it-works.mdx`

File: docs/processors/standalone/wasm.mdx (hunk 2 above)
Comment (lovromazgon) [reply docs/processors/standalone/wasm.mdx]: I simplified this header and renamed it to `WebAssembly` 😉

**Pull request merged**